### PR TITLE
[CI][BugFix] Remove e2e tracker guard on smart e2e

### PR DESCRIPTION
**Pull request by Potabk**
### What this PR does / why we need it?
This patch fixed an edge case where Smart e2e wasn't triggered as expected when only tests were added to the `tests/ut` directory. Ideally, Smart E2E shouldn't use E2E triggering conditions as its entry point. Furthermore, the specific routing logic should be left to Smart E2E to determine itself.
### How was this patch tested?

- vLLM version: v0.19.1
- vLLM main: https://github.com/vllm-project/vllm/commit/6f786f2c506cb07f4566771fdc62e640e2c4a176

### CODE REVIEW

**Review by leo-pony — APPROVED** [review]
LGTM

File: .github/workflows/pr_test_light.yaml [hunk 1]
```diff
@@ -87,7 +87,6 @@ jobs:
               - 'CMakeLists.txt'
 
       - name: Determine smart UT test scope
-        if: ${{ steps.filter.outputs.e2e_tracker == 'true' }}
```
Comment (leo-pony) [inline .github/workflows/pr_test_light.yaml:90]: 这里把ut_tracker为true添加上是不是就可以了？避免跳过现在的e2e-filter 的功能。
